src/components/Data/Documents/Page.vue

```diff
@@ -390,6 +405,41 @@ export default {
     }
   },
   methods: {
+    async subscribeToCurrentDocs() {
+      try {
+        if (this.subscribeRoomId) {
```

Comment:
NIT: You can refactor this to a dedicated function and avoid repeating this small chunk in the watcher and in this method